arm-network/2016-09-01/swagger/examples/RouteFilterCreate.json

```diff
@@ -0,0 +1,99 @@
+{
+  "parameters": {
+    "routeFilterName": "filterName",
+    "resourceGroupName": "rg1",
+    "api-version": "2016-12-01",
+    "subscriptionId": "subid",
+    "parameters": {
+      "location": "West US",
+      "tags": {
+        "key1": "value1"
+      },
+      "properties": {
+        "rules": [
+          {
+            "name": "ruleName",
+            "properties": {
+              "access": "Allow",
+              "routeFilterRuleType": "Community",
+              "communities": [
+                "12076:5030",
+                "12076:5040"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsofot.Network/routeFilters/filterName",
+        "etag": "w/\\00000000-0000-0000-0000-000000000000\\",
```

Comment:
[Resource definition](https://github.com/dihan0604/azure-rest-api-specs/blob/master/arm-network/2016-09-01/swagger/network.json#L97) referenced in the swagger spec does not have an "etag" property. Please add this to resource model definition.